tests/server.features/steps/langserver_steps.py

```diff
@@ -6,14 +6,16 @@
 # ----------------------------------------------------------------------------
 import tempfile
 from behave import given, when, then
-from coala_langserver.jsonrpc import TCPReadWriter
+from coala_langserver.jsonrpc import streams
 from coala_langserver.langserver import LangServer
 
+import logging
+logging.propagate = False
 
 @given('the LangServer instance')
 def step_impl(context):
-    context.f = tempfile.TemporaryFile()
-    context.langServer = LangServer(conn=TCPReadWriter(context.f, context.f))
+    context.file = tempfile.TemporaryFile()
```

Comment:
why does it *need* renaming from `f` to `file`?  It causes all sorts of other unnecessary changes to appear in this PR, slowing down the review.

is that related to the issue that this PR is intending to solve.

Reply:
'f' didn't seem to be a semantic name. My intention was to improve readability, retaining such occurrences could indicate tolerance for those practices and could even mean encouragement.

Reply:
I think the re-naming is justified, do you still recommend this particular change?

Reply:
Of course I do.  it is not related to using jsonrpc.  Create a new PR for other changes you wish to make.